src/ansys/fluent/core/fluent_connection.py

```diff
@@ -355,17 +375,32 @@ def force_exit_container(self):
         in that case use :func:`force_exit()`.
         If the Fluent session is responsive, prefer using :func:`exit()` instead.
         """
-        if not self.connection_properties.inside_container:
+        if self._remote_instance is not None:
             logger.error(
-                f"Session is not inside a container, cannot kill Fluent container. "
-                f"Use force_exit() instead."
+                f"Fluent is running remotely, cannot terminate Fluent container."
             )
             return
-        if self._remote_instance is not None:
-            logger.error(f"Fluent is running remotely, cannot kill Fluent container.")
+        container = self.connection_properties.inside_container
+        if not container:
+            logger.error(
+                f"Session is not inside a container, cannot terminate Fluent container. "
+                f"Try force_exit() instead."
```

Comment:
there are a few redundant f-strings in this module (already here)

Reply:
Thank you for the heads-up, going to double-check. Not sure why they ended up with a f-format specifier, probably due to changes over time and I forgot to remove them.